edgelet/api/identityServiceVersion_2020_06_01.yaml

```diff
@@ -0,0 +1,102 @@
+swagger: '2.0'
+schemes:
+  - http
+info:
+  title: Identity Service API
+  version: '2020-02-02'
+
+paths:
+  '/identities/identity':
```

Comment:
What do you think about /current-identity as the path?

Reply:
/identities/identity is a more REST-like name for the identity resource. It is also the name for the future IS API. Also, /current-identity has no meaning because it is always the identity of the caller. 

We are preserving /identities in the URL because we know there will be a backwards-compatible API that will be exposed for iotedged in the future.

Reply:
/identities/identity is not a standard REST like path. /identities/{id} is. 

To understand this API, aren't we returning result based on the caller?

Reply:
The caller doesn't know what {id} to pass in. It is querying for that {id} with /identities/identity.
How is /identities/identity not a valid REST path? If it has a documented contract for the resource - an identity spec - that is being returned, then any canonical name (i.e. identity) is valid here. Could you point to some example you have in mind when you say it is not standard?

Reply:
REST assumes resource for each API path. It encourages either plural or single. In general, folks use plural. 

Here the "/identities/identity" means the current identity of the caller. I see /currentIdentity is clearer to the caller and know what he/she is getting. 

Some of the examples:
https://docs.oracle.com/cd/E69533_01/Platform.11-3/ATGWSFrameGuide/html/s0519usingswaggerannotations01.html

https://app.swaggerhub.com/apis/Floktu/Floktu-API/3.0

Reply:
I understand the point. The current end of the path (/identity) is still a valid resource in the current proposal. The nesting of identity resource under /identities is the main concern in this discussion. I will change it to just /identity because current implies it is a current state (which cannot be verified by identity service alone because it doesn't actually connect - DPS tells it during provisioning and the client will need to connect to the cloud endpoint (Hub) with a token signed). Just authenticating with IS isn't enough for client to consider it a valid current identity. I will make this change and check in. That work?

Reply:
Sounds good. Thanks.